### PR TITLE
modfiy geo functions

### PR DESCRIPTION
**Pull request by yaojiefeng**

### CODE REVIEW

File: wrangler-core/src/main/java/io/cdap/functions/GeoFences.java [hunk 1]
```diff
@@ -47,7 +47,7 @@
    * @param geofences GeoJson representation of the fence area
    * @return true if location is inside any of the given geofences, else false
    */
-  public static Boolean inFence(double latitude, double longitude, String geofences) {
+  public static Boolean InFence(double latitude, double longitude, String geofences) {
```
Comment (albertshau) [inline wrangler-core/src/main/java/io/cdap/functions/GeoFences.java]: this is a backwards incompatible change, unless this method didn't exist before

Comment (yaojiefeng) [reply wrangler-core/src/main/java/io/cdap/functions/GeoFences.java]: Added back old methods and deprecated it.

File: wrangler-docs/functions/geo.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+### Input
+latitude(`double`), longitude(`double`), json fence (`string`)
+
+### Output
+true/false
+
+### Example
+if `latitude` contains `37.378990156513105` and `longitude` contains `-122.05076694488525` and
+`fence` contains a json `string`, then resulting operation is `true`
```
Comment (albertshau) [inline wrangler-docs/functions/geo.md]: contains a json string -> contains the previously mentioned json string

**Review by albertshau — APPROVED** [review]

**Pull request merged**